CallingWinUI/CallingWinUI.csproj

```diff
@@ -27,6 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.Communication.Calling.WindowsClient" Version="1.6.0" />
+	  <PackageReference Include="Microsoft.Toolkit.Uwp.Notifications" Version="7.1.3" />
```

Comment:
I think we decided not to show a sample of push notification in WinUI3 as it pretty much follows the same idea of UWP. And even the UWP one is not going to run out of the box as it needs a dev account to configure the whole thing e2e. So, I'd say we skip the PN sample in WinUI3 and focus on core ACS API and point developers to look at UWP sample to get an idea what it would take.